extensions/undertow/runtime/src/main/java/io/quarkus/undertow/runtime/ServletRuntimeConfig.java

```diff
@@ -26,4 +26,13 @@
     @ConfigItem
     Optional<Boolean> directBuffers;
 
+    /**
+     * The maximum number of HTTP request parameters permitted for Servlet requests.
+     *
+     * Corresponds to the
+     * <a href="https://undertow.io/undertow-docs/undertow-docs-2.1.0/index.html#common-listener-options">{@code MAX_PARAMETERS}
```

Comment:
I don't know if it makes sense to link to the Undertow 2.x docs here, as Quarkus HTTP is not the same thing.

Reply:
Done – replaced it with a sentence describing the effect.